### ISSUE TITLE
Deploy to an internal GSL VM

### ISSUE THREAD

**Issue by ian-noaa** [NOAA-GSL/vxDataProcessor#26]
## Problem

Now that we have a webserver, we want it running on an internal VM so MATS can make calls to it.

## Solution

Once #25 is done, create a trivial docker-compose file so that we can `docker-compose up -d` and `docker-compose down` on the VM.

## No Gos

_Describe any features or behaviors that have been considered and rejected as out of scope for this project._

We won't target Kubernetes yet.

## Rabbit Holes

_Describe any solutions to problems that pose a risk to completing this project on time._

**Comment by ian-noaa** [comment]
This has been done. The application is now responding on the `ascend-test1` VM. 

```console
$ curl ascend-test1.gsd.esrl.noaa.gov:8080/ping
{"message": "pong"}%
```
